Makefile

```diff
@@ -185,7 +195,7 @@ LIBC_TOP_HALF_ALL_SOURCES = \
     $(shell find $(LIBC_TOP_HALF_SOURCES) -name \*.c)
 
 # Set the target.
-CFLAGS = $(WASM_CFLAGS) --target=$(TARGET_TRIPLE)
+CFLAGS += --target=$(TARGET_TRIPLE)
```

Comment:
If someone overwrites `CFLAGS` on the make command line, does it override the `--target=$(TARGET_TRIPLE)` part too, or just the `-O2 -DNDEBUG` part?

Reply:
I think just the `O2 -DNDEBUG` part, but will check.

Reply:
@sunfishcode I tested it, and it indeed the CLI was overriding everything. I added a bunch of `override` so that this is no longer the case, and just the `-O2 -DNDEBUG` part gets overridden.